daemon/src/actor_name.rs

```diff
@@ -0,0 +1,3 @@
+pub trait ActorName {
+    fn actor_name() -> String;
+}
```

Comment:
I'd move this into `address_map` because that is where the code lives that motivates this piece. If we ever delete `address_map`, this should probably go away too!

If this assumption is no longer true at some point, we can move it.